setup.cfg

```diff
@@ -0,0 +1,21 @@
+[nosetests]
+verbosity=3
+with-doctest=1
+detailed-errors=1
+with-coverage=1
+cover-html=1
+cover-tests=1
+cover-erase=1
+cover-package=core_weekly
```

Comment:
I see you 👀 copying my code and stuff

Reply:
That was my code, it was late :sweat_smile:

Reply:
I know my friend ❤️ I'm just trolling :trollface: